source/style-guide/information-types/prototypes/reference/server-params-style.txt

```diff
@@ -0,0 +1,129 @@
+.. _meta-reference-server-params:
+
+=======================
+Server Parameters Style
+=======================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
```

Comment:
[Suggestion]

It looks a bit odd to me to jump immediately from the title to this h1. I'm wondering if we can consider adding a contextual sentence before the h1 explaining the overall goal of this page, or maybe even remove the h1 heading entirely.

Reply:
Removed. Thanks!